deploy/crds/kabanero.io_stacks_crd.yaml

```diff
@@ -55,6 +55,15 @@ spec:
                     items:
                       description: Image defines a container image used by a stack
                       properties:
+                        digest:
```

Comment:
I haven't looked at the rest of the code yet... but I think having the digest in the `spec` section is going to be confusing, and I'm guessing that we don't read it anywhere (or ignore it if it's there in the spec section).  I think that we need to think of a way to keep it out of the spec section.